forge-gui/res/cardsfolder/upcoming/callous_sell_sword_burn_together.txt

```diff
@@ -0,0 +1,22 @@
+Name:Callous Sell-Sword
+ManaCost:1 B
+Types:Creature Human Soldier
+PT:2/2
+K:etbCounter:P1P1:X:no Condition:CARDNAME enters the battlefield with a +1/+1 counter on it for each creature that died under your control this turn.
+SVar:X:Count$ThisTurnEntered_Graveyard_from_Battlefield_Creature.YouCtrl
+DeckHints:Ability$Graveyard
+DeckHas:Ability$Counters|Sacrifice
+AlternateMode:Adventure
+Oracle:Callous Sell-Sword enters the battlefield with a +1/+1 counter on it for each creature that died under your control this turn.
+
+ALTERNATE
+
+Name:Burn Together
+ManaCost:R
+Types:Sorcery Adventure
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | RememberObjects$ ThisTargetedCard | TgtPrompt$ Select target creature you control | SubAbility$ DBDamage | SpellDescription$ Target creature you control deals damage equal to its power to any other target. Then sacrifice it.
```

Comment:
@Agetian AILogic for Bite effects?

Reply:
@Simisays  `AILogic$ PowerDmg` see https://github.com/Card-Forge/forge/blob/master/forge-gui/res/cardsfolder/b/bite_down.txt